tools/build/configure.py

```diff
@@ -540,29 +707,49 @@ def build(object_paths: Union[Path, List[Path]], src_paths: List[Path], task: st
                     build(bin_path, [sprite_dir], "sprite", variables=variables)
                     build(yay0_path, [bin_path], "yay0")
                     build(
-                        self.build_path() / "include" / seg.dir / seg.name / (sprite_name + ".h"),
+                        self.build_path() / "include/sprite/npc" / (sprite_name + ".h"),
                         [sprite_dir],
                         "sprite_header",
                         variables=variables,
                     )
 
-                build(entry.object_path.with_suffix(".bin"), sprite_yay0s, "sprite_combine")
+                build(
+                    entry.object_path.with_suffix(".bin"),
+                    sprite_yay0s,
+                    "sprite_combine",
+                )
+                build(entry.object_path, [entry.object_path.with_suffix(".bin")], "bin")
+            elif seg.type == "pm_player_sprites":
+                build(
+                    entry.object_path.with_suffix(".bin"),
+                    entry.src_paths,
+                    "player_sprites",
+                    glob_deps=False,
+                )
```

Comment:
I think this needs implicit dependencies on all the files inside `entry.src_paths[0]` so that the player sprites are rebuilt when any of the source files change. Currently, I presume, this flow can happen:
1. Configure
2. Ninja
3. Modify `assets/*/sprite/player/SpriteSheet.xml`
4. "ninja: no work to do" 💀

Doing #1053 as discussed in my other comment should resolve this though, since we can make `entry.src_paths` be all the files instead of just the parent directory.

Reply:
I haven't set up the dependencies for the build tasks yet